include/small/lsregion.h

```diff
@@ -164,6 +172,28 @@ lslab_end(struct lslab *slab)
 	return (char *)slab + slab->slab_size;
 }
 
+/**
+ * A helper struct remembering a specific position in the lsregion. Is stable to
+ * lslab reuse.
+ */
+struct lsregion_svp {
+	/**
+	 * The max allocation id that was seen during a flush. Using it we can
+	 * find the slab we flushed previously and check if it has any extra
+	 * data to flush.
+	 */
+	int64_t max_id;
+	/** A position in the lslab containing an allocation with \a id. */
+	size_t pos;
+};
+
+static inline void
+lsregion_svp_create(struct lsregion_svp *svp)
```

Comment:
In obuf a similar method is called `obuf_svp_reset`.

Reply:
Yes, but obuf_svp also has a constructor: `obuf_create_svp`. In our case `lsregion_svp_create` is also a constructor.

If we take a look at other libraries, for example vclock or rlist, there are `vclock_create` and `rlist_create` which also bring the struct to the initial (empty) state. They are not `vclock_reset` (performs a different action) or `rlist_reset`(nonexistent).

I'd rather name this `lsregion_svp_create`, but I can change it if you like `lsregion_svp_reset` more.